### DISCUSSION TITLE
Error in downloading the project

### DISCUSSION THREAD

**Discussion by sophiastr** [ho-dev/HattrickOrganizer#1424] [General]
Hello, i am facing this problem. I tried to fixed it but i failed. Can you help me because i have an issue to solve...
![image](https://user-images.githubusercontent.com/72817947/160480628-f61a0908-19db-4c8f-ae35-0eb6fba1c741.png)

**Comment by tychobrailleur** [comment]
A way to get around it is to edit `gradle.properties`, and set `install4jHomeDir`:

```
install4jHomeDir = /tmp
```

**Comment by sophiastr** [comment]
Thank you, it worked!

**Comment by sophiastr** [comment]
One last question.(sorry)
I followed the steps and i commented the line referring to POEDITOR_APIKEY, but it doesn't work.
Any idea?
Thank you.
![image](https://user-images.githubusercontent.com/72817947/160490341-d52c6453-0885-43b2-aac5-5684eb997d3a.png)

**Reply by tychobrailleur** [reply]
> One last question.(sorry) I followed the steps and i commented the line referring to POEDITOR_APIKEY, but it doesn't work. Any idea? 

One trick I use when I work on HO is to comment the last 5 lines in `build.gradle`, as they tend to get in the way for me:

```
//installDist.dependsOn createLanguageFileList
//markdownToHtml.dependsOn pushmd
//poeditorPull.dependsOn markdownToHtml
//processResources.dependsOn poeditorPull
//compileJava.dependsOn processResources
```

The only thing is to ensure that this does not get committed into git.  This may not be appropriate if, for example, you are working on l10n, but is ok most of the time.

Another thing you may want to do is to preserve your local DB – I usually comment out the deletion of `db` local folder, and comment out content of `resetDB`, as seeing this being deleted all the time is pet peave of mine.

Hope this helps.

**Comment by sophiastr** [comment]
Thank you very much. Problem solved. Thanks
